core/test/dummy/config/application.rb

```diff
@@ -16,6 +16,6 @@ class Application < Rails::Application
     # Settings in config/environments/* take precedence over those specified here.
     # Application configuration should go into files in config/initializers
     # -- all .rb files in that directory are automatically loaded.
-    config.time_zone = 'Eastern Time (US & Canada)'
+    config.time_zone = 'Sydney'
```

Comment:
I know you were doing this for testing, but should this always be set in the dummy files? Do you think it will help catch more errors like this?

Reply:
Maybe. I thought about possibly having an ENV we could set in builds and then using the matrix functionality to get builds running with different timezones set. We can do that if it comes up again.